package.json

```diff
@@ -60,6 +60,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.22.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
+    "css-loader": "^0.27.3",
```

Comment:
We can remove `css-loader` from the root package.json. This dependency is only used in webpack so we should only need this dependency in terra-site.